### PR TITLE
Revamp readme

### PR DESCRIPTION
**Pull request by stelios-ritual**
## Motivation

- Revame README
- Update versions
- Anvil works out of the box

### CODE REVIEW

**Review by evangelos-ritual — COMMENTED** [review]
Great!

Also on the main branch, container id names should adhere to `[a-zA-Z0-9][a-zA-Z0-9_.-]+`

**Review by evangelos-ritual — COMMENTED** [review]
- After adding a service,

In node version 1.2.0, `"port": 3000` is missing

In node version 1.3.0 the empty config params produce error:
`[error    ] Config validation error: field 'accepted_payments'. Input should be a valid dictionary [shared.container] `

- When trying to specify version `infernet-cli config anvil -v "1.3.0"` it allows only for 1.3.0 but the node image is always 1.2.0 in docker compose

**Comment by stelios-ritual** [comment]
Please note, recipes / CLI are only supposed to be compatible with node v1.3.0+, for that exact reason (auto-assigning ports etc.)

> In node version 1.3.0 the empty config params produce error:
[error    ] Config validation error: field 'accepted_payments'. Input should be a valid dictionary [shared.container] 
fixed

> When trying to specify version infernet-cli config anvil -v "1.3.0" it allows only for 1.3.0 but the node image is always 1.2.0 in docker compose
fixed

@evangelos-ritual

File: services/hf-client-inference/1.0.0/recipe.json [hunk 1]
```diff
@@ -1,10 +1,18 @@
 {
     "config": {
-        "id": "hf-client-inference:1.0.0",
+        "id": "hf-client-inference-1.0.0",
         "image": "ritualnetwork/hf_client_inference_service:1.0.0",
```
Comment (evangelos-ritual) [inline services/hf-client-inference/1.0.0/recipe.json:4]: Monorepo name and image name is `hf_inference_client_service`

**Review by evangelos-ritual — COMMENTED** [review]
Great, ran it across all networks, + added css_service and hf_service, works fine!

File: services/hf-client-inference/1.0.0/recipe.json (hunk 1 above)
Comment (stelios-ritual) [reply services/hf-client-inference/1.0.0/recipe.json]: good catch

**Review by evangelos-ritual — APPROVED** [review]

**Pull request merged**